db/views/rejection_percentage_by_days_v01.sql

```diff
@@ -0,0 +1,69 @@
+WITH rejection_reasons AS (
```

Comment:
I'm not married to a particular style, but would like to keep the style consistent across files.